### ISSUE TITLE
IHE HQ changed address?

### ISSUE THREAD

**Issue by JohnMoehrke** [IHE/publications#450]
It seems that www.ihe.net now declares the HQ is
1502 W. Broadway, Suite 102, Madison, WI 53713

the profiles.ihe.net still has the RSNA address... what should profiles.ihe.net have?
- Should profiles.ihe.net not have any postal address? It links to www.ihe.net for that kind of thing
- Should it be updated?

**Comment by MaryLJ** [comment]
I talked to Chris Carr about this as we should update the address on profiles.ihe.net as well. We should remove the phone number as well as that is not listed on ihe.net (if it was before, it has been removed).

**Comment by JohnMoehrke** [comment]
This will change EVERY page. We should do this independent of specification releases. So lets do this as a bulk change after we release the PCC/ITI stuff, and after we tag that release. 

I wonder if we could use a variable, so that when it changes again we can just change the variable in the one CSS file.

**Comment by MaryLJ** [comment]
OK. I will check on using a variable, that would make sense.

 

Mary

 

Mary Jungers | Documentation Specialist 

IHE International | IHE.net

**Comment by JohnMoehrke** [comment]
It seems possible, but not as easy as I had hoped. Might be able to talk to someone who understands CSS -> HTML better than me (not hard to find them) and have someone provide a solution. 

Note that if we do the above, we can't use that in the IG publisher or even the long-form publications.. so ultimately we will still end up with lots of copies of the address everywhere.    So, we might just do it the hard way, and bulk change, thus not using a CSS variable.

**Comment by MaryLJ** [comment]
OK. I will check with Chris if it could be removed as an option, too.

 

Mary

 

Mary Jungers | Documentation Specialist 

IHE International | IHE.net

**Comment by JohnMoehrke** [comment]
agreed. is it really useful to have a physical address on a web page. When this is the kind of thing that the ABOUT page should focus on.  I suspect it is a hold over from the print days.

**Comment by MaryLJ** [comment]
Per a conversation with Chris Carr, the phone number and address can be removed (it has changed again). So at a convenient time, we can make the change for the footer just to be "© 2000 — 2025 IHE International"

**Comment by JohnMoehrke** [comment]
Do we need to include 2025? That means we must change every page again every year for no good reason.

**Comment by MaryLJ** [comment]
> Do we need to include 2025? That means we must change every page again every year for no good reason.

Agree, that we should not include 2025

**Comment by MaryLJ** [comment]
> Now I hear the Janesville address is the right one. https://www.ihe.net/about_ihe/contact_us/

We should remove it the next time we have an opportunity to update the footer.
